web/themes/custom/usagov/usagov.theme

```diff
@@ -10,13 +10,167 @@ use Drupal\taxonomy\Entity\Term;
  * Place your custom PHP code in this file.
  */
 
-/**
-* This makes the node varialbe available to the templates/html.html.twig template file
-*/
 function usagov_preprocess_html(&$variables) {
-  $variables['node'] = \Drupal::routeMatch()->getParameter('node');
+  $node = \Drupal::routeMatch()->getParameter('node');
+  $variables['node'] = $node;
+
+  // If the FAQ checkbox is checked, then parse the content to find questions and answers
+  if ($node && $node->hasField('field_faq_page') && $node->get('field_faq_page')[0]->value) {
```

Comment:
I'm still getting an error message here on the home page. It turns out that on the home page, 'field_faq_page" exists but it's an empty array. So (ugh), 

`if ($node && $node->hasField('field_faq_page') && $node->get('field_faq_page')[0] && $node->get('field_faq_page')[0]->value) {`

This did solve my login problem, though!

Reply:
I tried using the nullsafe operator like this and it seems to do the trick:
`if ($node?->hasField('field_faq_page') && $node->get('field_faq_page')[0]?->value) {`